upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.12.yaml.template

```diff
@@ -616,37 +810,16 @@ spec:
             # Disable IPv6 on Kubernetes.
             - name: FELIX_IPV6SUPPORT
               value: "false"
-            # Set Felix logging to the desired level
+            # Set Felix logging to "info"
             - name: FELIX_LOGSEVERITYSCREEN
-              value: "{{- or .Networking.Calico.LogSeverityScreen "info" }}"
```

Comment:
This looks to be the last remaining use of this variable in which case it's probably worth removing it from the `CalicoNetworkingSpec` in `networking.go` to avoid confusion.

Reply:
Updated! Thanks!